### PR TITLE
Add PBR shader tutorial guide

### PR DESCRIPTION
**Pull request by TheChunkierBean**
A written guide that showcases how to port a PBR texture set to Halo 3.

### CODE REVIEW

**Review by num0005 — CHANGES_REQUESTED** [review]
Please setup a page.yml see https://github.com/Sigmmma/c20/blob/master/src/content/h2/guides/map-making/level-creation/page.yml for an example

**Review by num0005 — CHANGES_REQUESTED** [review]
you can also remove this since the page.yml handles this already.

**Comment by num0005** [comment]
I will review the content later but that should be all the technical issues.

**Comment by TheChunkierBean** [comment]
Removed acknowledgements from readme.md

File: src/content/h3/guides/shaders/pbr-shader/readme.md [hunk 1]
```diff
@@ -15,6 +15,9 @@ Physically based rendering (PBR) is a way of rendering images that mimics how li
 
 With the release of the weapon and vehicle skins update to Halo 3 MCC, 343 Industries implemented a new material_model called **cook_torrance_pbr_maps**. This allowed for the menu skins that were authored in PBR to be replicated in-engine when enabled. While not 100% true PBR, this shader allows artists to now have a more modern rendering technique in Halo 3.
 
+
+**NOTE:** *There is no definitive way to utilize this shader at this time, as even 343 Industries uses inconsistent shader settings with this shader. This guide however is a good baseline setup to start with.*
+
```
Comment (num0005) [inline src/content/h3/guides/shaders/pbr-shader/readme.md:20]: Please use the `.alert` syntax for this (see other pages for an example of how to do it.

File: src/content/h3/guides/shaders/pbr-shader/page.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+title:
+  en: H3 pbr shader guide
```
Comment (num0005) [inline src/content/h3/guides/shaders/pbr-shader/page.yml:2]: capitalize PBR
```suggestion
  en: H3 PBR shader guide
```

File: src/content/h3/guides/shaders/pbr-shader/readme.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+* ![](A.png "It's like talking to a brick wall")
+
+Halo 3 includes a PBR-like shader that allows artists to convert their premade PBR assets to work with Halo 3's shader system. This guide will explain how to convert your existing PBR assets into Halo 3. 
+
+# What is PBR?
+
+* ![](B.png "An example of PBR")
+
+Physically based rendering (PBR) is a way of rendering images that mimics how light works in the real world. Most modern game engines have a form of this method in there shading systems.
```
Comment (num0005) [inline src/content/h3/guides/shaders/pbr-shader/readme.md:10]: Maybe add a wikipedia link here?

File: src/content/h3/guides/shaders/pbr-shader/readme.md [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+4. Set the blend mode of the Ambient Occulsion to **Multiply**.
+  * ![](E.png "Multiplied Ambient Occulsion map")
+5. Save your new Diffuse map as .TIF in your asset's bitmaps folder.
+
+You now have a converted Diffuse map. Make sure to save a editable copy of the Diffuse map in case you wish to make changes later.
+
+
+# Step 2: Creating a Specular map
+
+**NOTE**: *If you have a texture set that does not include a **Metallic map**, you do not need to make a Specular map. You can skip this step.*
```
Comment (num0005) [inline src/content/h3/guides/shaders/pbr-shader/readme.md:56]: ditto

File: src/content/h3/guides/shaders/pbr-shader/readme.md [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+  * ![](J.png "Roughness map in the green channel")
+7. Save your new Control map as .TIF in your asset's bitmaps folder.    
+
+The final Control map should resemble a pink and blue texture, with the pink colors representing the reflective surfaces and the blue colors the dull surfaces. If you have a material that has no metal, the Control map should only have blue colors. 
+
+* ![](K.png "Final Control map")
+
+# Step 4: Converting a Normal Map
+
+Make sure to identify what type of Normal map you have. Halo 3 uses DirectX normals and those do not need to be converted to work with the engine. Simply save the normal as a .TIF in your asset's bitmaps folder and you will be good to go.
```
Comment (num0005) [inline src/content/h3/guides/shaders/pbr-shader/readme.md:109]: maybe link to something that explains what the difference between those two is? Or would it be obvious to anyone reading this guide?

**Review by num0005 — APPROVED** [review]
Looks fine apart from the few things pointed out.

**Review by num0005 — APPROVED** [review]

**Pull request merged**